app/forms/contributor_populator.rb

```diff
@@ -10,7 +10,7 @@ def call(form, fragment:, **)
     item = existing_record(form: form, id: fragment['id'])
 
     if fragment['_destroy'] == '1'
-      form.contributors.delete(item)
+      form.send(field).delete(item)
```

Comment:
No strong feelings about this, but since `ApplicationPopulator` already has `#value`, this could also be: `value(form).delete(item)` Alternatively, consider changing `send` to `public_send`?

Reply:
Good eye. I have made those changes.